packages/ecc-client-ga4gh-tes/src/data/baseURL.ts

```diff
@@ -0,0 +1,10 @@
+const baseURL = "https://csc-tesk-noauth.rahtiapp.fi/v1";
```

Comment:
This will be provided by the user using the component

Reply:
passed as attr ? 
`<comp url="url"></comp>`

Reply:
Yes @uniqueg can confirm!